backend/application/favoriteVideo.ts

```diff
@@ -0,0 +1,71 @@
+import { PrismaClient } from '@prisma/client'
+
+const prisma = new PrismaClient()
+
+// type FavoriteVideoType = {
+//   video_id: string
+//   status: boolean
+// }
```

Comment:
使う予定なら

```suggestion
// TODO　追加予定
// type FavoriteVideoType = {
//   video_id: string
//   status: boolean
// }
```

みたいに何かコメントしておいてもらえると

Reply:
@issiyda 
こちらは消し忘れです。失礼ました…！
削除しておきました。